subsys/net/l2/openthread/openthread.c

```diff
@@ -299,42 +299,17 @@ static enum net_verdict openthread_recv(struct net_if *iface,
 
 int openthread_send(struct net_if *iface, struct net_pkt *pkt)
 {
-	struct openthread_context *ot_context = net_if_l2_data(iface);
 	int len = net_pkt_get_len(pkt);
-	struct net_buf *buf;
-	otMessage *message;
-	otMessageSettings settings;
 
 	NET_DBG("Sending Ip6 packet to ot stack");
```

Comment:
This debug is also present in `openthread_handle_frame_to_send` now, IMO it's enough to have it in one place.